resources/install/debian/control

```diff
@@ -3,15 +3,14 @@ Section: net
 Priority: extra
 Maintainer: Jitsi Team <dev@jitsi.org>
 Uploaders: Emil Ivov <emcho@jitsi.org>, Damian Minkov <damencho@jitsi.org>
-Build-Depends: debhelper (>= 8.0.0)
+Build-Depends: debhelper (>= 9), dh-systemd
 Standards-Version: 3.9.3
 Homepage: https://jitsi.org/videobridge
 
 Package: jitsi-videobridge
 Architecture: any
 Pre-Depends: openjdk-8-jre-headless
-Depends: ${misc:Depends}, procps
-Recommends: authbind
+Depends: ${misc:Depends}, procps, systemd (>= 29.1)
```

Comment:
The current version (upstream) is v239. Isn't this dependency ancient?

Reply:
Well, I saw it at some place with examples ... I started working on this maybe two months ago and I do not remember what was the reason for this ... there was something not supported before this version. But I think to just remove it then, and leave systemd requirement without version, WDYT?

Reply:
Sounds good!